app/touch-bar-menu.js

```diff
@@ -3,7 +3,7 @@
 const { TouchBar } = require('electron')
 
 const { TouchBarButton, TouchBarLabel, TouchBarSpacer } = TouchBar
-const MAX_TITLE_LENGTH = 38
+const MAX_TITLE_LENGTH = 35
```

Comment:
@salomvary depending on the track playing this might have to be even lower. I think it depends on the width of the string once drawn. If it's too wide it doesn't get drawn at all.